src/components/Database/DataViewer/ValueDisplay.tsx

```diff
@@ -37,7 +37,7 @@ export const ValueDisplay = React.memo<Props>(function ValueDisplay$({
     `ValueDisplay ValueDisplay__${typeof value}`;
   return (
     <Typography use="body1" className={getClass(value)} {...otherProps}>
-      {JSON.stringify(trimmedValue)}
+      {value === null ? '{}' : JSON.stringify(trimmedValue)}
```

Comment:
Let's still show it as `null`, since that's what you'll get back from SDK and REST apis. I don't want to see a bug report highlighting the disparency.

Reply:
To elaborate, the way we usually document and explain this goes: There is no actual `{}` in RTDB, only `null`. There's an RTDB feature that creates an object for you if you add a field to `null`, which is working as intended. And if you delete all fields in an object, it turns back into `null`.